packages/serialization/json/package.json

```diff
@@ -31,17 +35,6 @@
         "url": "https://github.com/microsoft/kiota-typescript/issues"
     },
     "homepage": "https://github.com/microsoft/kiota-typescript#readme",
-    "devDependencies": {
-        "@types/node": "^17.0.22",
-        "@typescript-eslint/eslint-plugin": "^5.16.0",
-        "@typescript-eslint/parser": "^5.16.0",
-        "eslint": "^8.11.0",
-        "eslint-config-prettier": "^8.5.0",
-        "eslint-plugin-prettier": "^4.0.0",
-        "eslint-plugin-simple-import-sort": "^7.0.0",
-        "prettier": "^2.6.0",
-        "typescript": "^4.6.2"
-    },
     "dependencies": {
         "@microsoft/kiota-abstractions": "^1.0.0-preview.1",
```

Comment:
shouldn't this dep be tokenized by lerna?

Reply:
lerna versioning takes care of it. 

The command lerna version updates the dependencies